Age-ify.js

```diff
@@ -0,0 +1,6 @@
+//A future age calculator
+
+const yearOfBirth = 1989;
+const yearFuture = 2045;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".")
```

Comment:
This could be a bit more tidy with a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) syntax. Give it a try.

Reply:
Thank you for the feedback, fixed.